homeassistant/components/sensor/yr.py

```diff
@@ -111,7 +111,7 @@ def entity_picture(self):
         """Weather symbol if type is symbol."""
         if self.type != 'symbol':
             return None
-        return "http://api.met.no/weatherapi/weathericon/1.1/" \
+        return "https://api.met.no/weatherapi/weathericon/1.1/" \
```

Comment:
Would probably be best to use the `//` scheme, so that it uses whatever scheme the page was loaded with.

i.e. `//api.met.no/weatherapi/weathericon/1.1/`